features/step_definitions/additional_cli_steps.rb

```diff
@@ -82,6 +82,24 @@
   expect(all_output).to include("\e[35m" + "F" + "\e[0m")
 end
 
+Then /^the failed example includes shell hyperlink escape codes$/ do
+  open = "\e\\]8;;"
+  divider = "\e\\\\"
+  close = "\e\\]8;;"
+  uri = "rspec://.*/hyperlinks_spec.rb:6"
+  text = "./hyperlinks_spec.rb:6"
+  expect(all_output).to match(Regexp.new(open + uri + divider + text + close))
+end
+
+Then /^the failed shared examples includes shell hyperlink escape codes$/ do
```

Comment:
These are badly named, something more along the lines of "includes properly escaped links to files"